src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/LastKnownAddressRepository.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.db
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.repository.NoRepositoryBean
+import org.springframework.stereotype.Component
+import org.springframework.stereotype.Repository
+import java.util.UUID
+
+@Repository("jpaLastKnownAddressRepository")
+interface JpaLastKnownAddressRepository : JpaRepository<LastKnownAddress, UUID>
+
+@NoRepositoryBean
+interface ExtendedLastKnownAddressRepository : JpaLastKnownAddressRepository
+
```

Comment:
Never sure if we use these anymore. Recall Mike adding them for some tests 🤷‍♂️

Reply:
🤷‍♂️